packages/webpack-cli/lib/webpack-cli.js

```diff
@@ -91,6 +91,15 @@ class WebpackCLI extends GroupHelper {
     getCoreFlags() {
         return core;
     }
+
+    /**
+     * Convert camelcase to kebabcase
+     * @param {string} string
+     */
+    _toKebabCase(string) {
```

Comment:
I think any helper function should live in a utils file, instead of being a part of the class